packages/web3-providers-base/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { ProviderOptions } from '../types';
+
+export default class Web3ProviderBase {
+  private _providerString: string
```

Comment:
nit: `_providerUrl` makes more sense?

Reply:
Would you call it a URL for IPC as well?

Reply:
Technically you can think of the filesystem as a url `ipc:///` or `file:///`

Reply:
Updated `providerString` to `providerUrl`